.github/workflows/ci.yml

```diff
@@ -59,12 +59,11 @@ jobs:
             test_flags: "--platform=x64 --config=Debug"
             msbuild_project: "msbuild/ice.proj"
 
-          # Swift iOS
+          # iOS
           - os: macos-15
-            config: "swift-ios"
-            working_directory: "swift"
-            build_flags: "PLATFORMS='macosx iphonesimulator'"
-            test_flags: "--platform=iphonesimulator --controller-app --rfilter=Ice/operations" # Remove rfilter once #3286 is fixed
+            config: "ios"
+            build_flags: "PLATFORMS='macosx iphonesimulator' LANGUAGES='cpp swift' CONFIGS='static'"
+            test_flags: "--platform=iphonesimulator --controller-app --rfilter=swift/Ice/operations" # Remove rfilter once #3286 is fixed
```

Comment:
Do we need to do `--filter=cpp` `--filter=swift`

Reply:
I think we don't as there no other languages that support `iphonesimulator` platform.

Reply:
Added `--languages='cpp,swift'`